tests/acl/test_acl.py

```diff
@@ -860,11 +865,21 @@ def _verify_acl_traffic(self, setup, direction, ptfadapter, pkt, dropped, ip_ver
 
         ptfadapter.dataplane.flush()
         testutils.send(ptfadapter, self.src_port, pkt)
-
-        if dropped:
-            testutils.verify_no_packet_any(ptfadapter, exp_pkt, ports=self.get_dst_ports(setup, direction))
+        if direction == "uplink->downlink":
+            dst_port = DOWNSTREAM_IP_PORT_MAP.get(pkt[packet.IP].dst)
+            pytest_assert(dst_port != None,
+                          "Can't find dst port for IP {}".format(pkt[packet.IP].dst))
+
```

Comment:
How about move this logic into `get_dst_ports`?

Reply:
Yes, should be better.